File: SkoloInstituteAPI/SkoloInstitute/Entities/DataTransferObjects/Rating/RatingDto.cs
namespace SkoloInstitute.Entities.DataTransferObjects.Rating
{
    public class RatingDto
    {
        public Guid Id { get; set; }
        //  public string UserId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string ReviewBody { get; set; }
        public DateTime CreatedDate { get; set; }
        public int starsCount { get; set; }
    }
}

File: skolo-ui/src/app/_interfaces/rating/RatingDto.ts
export interface RatingDto {
    firstName: string;
    lastName: string;
    reviewBody: string;
    createdDate: Date;
    starsCount: number;
}

export interface RatingForCreationDto {
  userId?: string;
  reviewBody: string;
  starsCount: number;
  teacherId: string; 
}
